sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -48,6 +48,7 @@
   }
 
   // TODO: Should be a Stream
+  @SuppressWarnings("NullAway")
```

Comment:
What is it warning about exactly? And why do we ignore?
```suggestion
  @SuppressWarnings("NullAway") // Suppress because ...
```

Reply:
`dataFactory` is a `Callable` which returns theoretically a nullable result. Our implementations always provide non-null values. We can handle potential `null`, but how to handle it? Log and return an empty byte array from this method?

Reply:
If I read it right we currently return an empty byte array.

The note was mainly to suggest adding a comment:

```suggestion
  // dataFactory is a Callable which returns theoretically a nullable result. Our implementations always provide non-null values. 
  // In the even that it did return null, we compensate by returning an empty byte array (and logging an error)
  @SuppressWarnings("NullAway")
```